packages/manager/apps/pci-workflow/src/components/new/ResourceSelector.component.tsx

```diff
@@ -65,34 +65,22 @@ const useDatagridColumn = (
       id: 'name',
       cell: (instance: TWorkflowInstance) => (
         <NotSupportedTooltipComponent region={instance.region}>
-          <DataGridTextCell
-            className={isLocalZone(instance.region) ? 'opacity-50' : ''}
-          >
-            {instance.name}
-          </DataGridTextCell>
+          <DataGridTextCell>{instance.name}</DataGridTextCell>
         </NotSupportedTooltipComponent>
       ),
       label: t('pci_projects_project_workflow_instance_name_label'),
     },
     {
       id: 'region',
       cell: (instance: TWorkflowInstance) => (
-        <DataGridTextCell
-          className={isLocalZone(instance.region) ? 'opacity-50' : ''}
-        >
-          {instance.regionName}
-        </DataGridTextCell>
+        <DataGridTextCell>{instance.regionName}</DataGridTextCell>
       ),
       label: t('pci_projects_project_workflow_instance_region_label'),
     },
     {
       id: 'flavorName',
       cell: (instance: TWorkflowInstance) => (
-        <DataGridTextCell
-          className={isLocalZone(instance.region) ? 'opacity-50' : ''}
-        >
-          {instance?.flavorName}
-        </DataGridTextCell>
+        <DataGridTextCell>{instance?.flavorName}</DataGridTextCell>
```

Comment:
Is it normal that you are using optional chaining only in this DatagridTextCell ?